app/assets/javascripts/initializers/initializeBaseTracking.js.erb

```diff
@@ -111,6 +111,9 @@ function trackCustomImpressions() {
 
     // page view
     if (ArticleElement && tokenMeta && !isBot) {
+      // See https://github.com/forem/forem/blob/05413f101ab92eb7e25cf5b098919e21179be602/app/controllers/page_views_controller.rb
```

Comment:
This link is going to a particular commit, should we link to the current `main` branch for this file instead?
```suggestion
      // See https://github.com/forem/forem/blob/main/app/controllers/page_views_controller.rb
```